README.md

```diff
@@ -100,47 +101,58 @@ $ nix run github:Davidson-Souza/Floresta
 
 After building, florestad and floresta-cli will be available in the target directory. You can run the full node with
 ```bash
-./target/release/florestad
+$ ./target/release/florestad
 # or, if you installed it with cargo install
-florestad
+$ florestad
 ```
 
 This will start the full node, and you can connect to it with an Electrum wallet or with the `floresta-cli` tool.
 
 ```bash
-floresta-cli getblockchaininfo
+$ floresta-cli getblockchaininfo
 ```
 
 For more information on how to use the `floresta-cli` tool, you can check the [api documentation](https://github.com/Davidson-Souza/Floresta/blob/master/crates/floresta-cli/README.md).
 
 #### Assume Utreexo
+
 If you want to skip the IBD process, you can use the `--assumeutreexo` flag. This flag will start the node at a given height, with the state
 provided by this implementation. Therefore, you're trusting that we are giving you the correct state. Everything after that height will be
 verified by the node just like any other node.
 
 ```bash
-florestad --assumeutreexo
+$ florestad --assumeutreexo
+```
+
+#### Backfill
+
+If you want to get a node up and running in a few minutes, but still want to validate everything, you can start `florestad` with `assumeutree` and pass the `--backfill` flag. This will download the blocks from genesis to the assumed height, validate them and compare with the provided value, all in the background. This way, you can start using the node right away, and it will be fully validated in a few hours/days.
+
```

Comment:
```suggestion
If you want to get a node up and running in a few minutes, but still want to validate everything, you can start `florestad` with `assumeutreexo` and pass the `--backfill` flag. This will download the blocks from genesis to the assumed height, validate them and compare with the provided value, all in the background. This way, you can start using the node right away, and it will be fully validated in a few hours/days depending on your hardware.
```